storage/replica_command.go

```diff
@@ -2336,6 +2336,9 @@ func (r *Replica) AdminSplit(
 			return err
 		}
 		if err := txn.Run(b); err != nil {
+			if _, ok := err.(*roachpb.ConditionFailedError); ok {
+				return errors.Errorf("conflict updating range descriptors")
```

Comment:
Assuming that that message contained some more context, I'd have a hard time figuring out when I would actually need the extra bit. I had #7600 and comrades in the back of my mind. Should be fine to omit this, though.